Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
```

Comment:
We generally haven't been for other kapost gems, unless there's a reason for it that I'm not aware of.  I guess even if you do commit it, it doesn't get packaged at the time that you `bundle release` it.